src/Trips.js

```diff
@@ -0,0 +1,26 @@
+class Trips {
+  constructor(tripsData, destinationData) {
+    this.trips = tripsData;
+    this.destinations = destinationData;
```

Comment:
You sure you want these two arrays on the same class? It might be easier, I suppose, but you also might be setting yourself up for some complicated methods!